pages/about.html

```diff
@@ -0,0 +1,154 @@
+      </div>
+      
+      <div class="heroContent contactEmail">
+        <p>Please contact us on our various platforms for any further questions about our programs for 2023! </p>
+        <a href="mailto:ginoschool@gmail.com? subject=Enquiry for Admission">ginoschool@gmail.com</a>
+      </div>
+      <div id="about"></div>
+    </section>
+
+    <!-- ========= About School Section ===================== -->
+
+    <section class="schoolInfo">
+      <div class="schoolDetails">
+        <h2>About our <br> Videography School</h2>
+        <hr>
+        <p>Our videography school stands at the forefront of innovation, empowering aspiring videographers to capture the world through a unique blend of technical mastery and artistic expression.</p>
+      </div>
+      <div class="footer aboutSchool">
+        <div class="footerLogo"><a href="#"><img src="../assets/logo1.png" alt=""></a>
+          <p id="about-description">The Excellence of <br>videography</p>
+        </div>
+        <div id="graduates"></div>
+      </div>
+    </section>
+
+    <!-- =================== Graduates Section =================== -->
+
+    <section class="schoolInfo">
+      <div class="schoolDetails">
+        <h2>Celebrating our Accomplished Graduates</h2>
+        <hr>
+        <p>Our videography school produces remarkable graduates who excel in capturing unforgettable moments and telling compelling visual stories, leaving a lasting impact in the industry.</p>
+      </div>
+
+      <div class="graduatesContainer">
+        <div class="graduatesCard card1">
+          <div class="graduatesContent">
+            <h2>2022</h2>
+            <p>Celebrating the journey of our talented graduates.</p>
+          </div>
+        </div>
+        <div class="graduatesCard card2">
+          <div class="graduatesContent">
+            <h2>2021</h2>
+            <p>Exploring new horizons with our successful alumni.</p>
+          </div>
+        </div>
+      </div>
+      
+      
+    </section>
+
+ <!-- ========== Sponsor ================== -->
+
+    <section class="sponsorContainer" id="sponsor">
+      <div class="sponsorHeading">
+        <h2>Sponsor</h2>
+        <hr>
+      </div>  
+    <div class="sponsorGrid">
+      <div class="sponsorCard">
+        <i class="fa-solid fa-blog fa-xl"></i><p>Blog</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-google fa-xl"></i><p>Google</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-joomla fa-xl"></i><p>Joomla</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-stumbleupon fa-xl"></i><p>Stumbleupon</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-yahoo fa-xl"></i><p>Yahoo</p>
+      </div>
+    </div>
+    
+    </section>
+
+  </main>
+
+  <footer>
```

Comment:
As was required by the previous reviewer

- Kindly change the background color of your footer so it will match with Behance design 💯